tests/drivers/opamp/opamp_api/testcase.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+common:
+  tags:
+    - drivers
+    - opamp
+    - userspace
```

Comment:
Please double-check regarding user space. My implementation used it, cause I've provided syscals to user space. Your implementation was agreed not to provide user space calls, since it makes les sense for opamp.